nodewatcher/settings.py

```diff
@@ -452,6 +452,7 @@ def user_url(user):
     'nodewatcher.modules.monitor.http.resources.processors.SystemStatus',
     'nodewatcher.modules.monitor.http.interfaces.processors.DatastreamInterfaces',
     'nodewatcher.modules.monitor.http.clients.processors.ClientInfo',
+    'nodewatcher.modules.monitor.http.survey.processors.SurveyInfo',
```

Comment:
It is already included in `MONITOR_RUNS`, check the `telemetry` and `telemetry-push` run definitions, which reference `TELEMETRY_PROCESSOR_PIPELINE`.